seed/csharp-sdk/alias-extends/src/SeedAliasExtends/Core/JsonConfiguration.cs

```diff
@@ -100,6 +99,58 @@ internal static JsonDocument SerializeToDocument<T>(T obj) =>
     internal static byte[] SerializeToUtf8Bytes<T>(T obj) =>
         JsonSerializer.SerializeToUtf8Bytes(obj, JsonOptions.JsonSerializerOptions);
 
+    internal static string SerializeWithAdditionalProperties<T>(
+        T obj,
+        object? additionalProperties = null
+    )
+    {
+        if (additionalProperties == null)
+        {
+            return Serialize(obj);
+        }
+        var additionalPropertiesJsonNode = SerializeToNode(additionalProperties);
+        if (additionalPropertiesJsonNode is not JsonObject additionalPropertiesJsonObject)
+        {
+            throw new InvalidOperationException(
+                "The additional properties must serialize to a JSON object."
+            );
+        }
+        var jsonNode = SerializeToNode(obj);
+        if (jsonNode is not JsonObject jsonObject)
+        {
+            throw new InvalidOperationException(
+                "The serialized object must be a JSON object to add properties."
+            );
+        }
+        MergeJsonObjects(jsonObject, additionalPropertiesJsonObject);
+        return jsonObject.ToJsonString(JsonOptions.JsonSerializerOptions);
+    }
+
+    private static void MergeJsonObjects(JsonObject baseObject, JsonObject overrideObject)
+    {
+        foreach (var property in overrideObject)
+        {
+            if (!baseObject.ContainsKey(property.Key))
```

Comment:
In C# we use the TryX if available:
[TryGetPropertyValue(String, JsonNode)](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.nodes.jsonobject.trygetpropertyvalue?view=net-9.0#system-text-json-nodes-jsonobject-trygetpropertyvalue(system-string-system-text-json-nodes-jsonnode@))